### DISCUSSION TITLE
Frame accurate sync between two GoPro Hero 10 with 'The Remote' for 3D photography issues

### DISCUSSION THREAD

**Discussion by uhdtv3d** [gopro/labs#1137] [Feature Requests]
I am using two GoPro Hero 10s with identical settings to capture stereoscopic 3D stills. I use GoPro's 'The Remote' to trigger both cameras simultaneously. However, I’ve noticed they don’t sync perfectly like a dedicated 3D camera. This discrepancy makes it challenging to capture still photos of subjects effectively, even when their movement is very subtle!

Is there any existing way to sort it out?

**Comment by BobVe** [comment]
Just a suggestion, you could try shooting short clips in video mode, exporting as image sequences, and try to find matching frames. 

I shoot 3D video at high frame rates (240 fps) with Hero11 cameras. Sometimes sync is excellent, once clips have been matched up. I use a combination of photoshop, bridge, and a file renaming program. It can be tedious, but I can get great image pairs. 

HTH
